insilicho/run.py

```diff
@@ -87,23 +87,28 @@ def _randomize_params(self, rel_stddev: float):
             rel_stddev (float): Relative std deviation while sampling parameter, assumes
                 a normal distribution.
         """
-        float_params = {
+        noisy_params = self.params_with_noise()
+        for pname, pval in noisy_params.items():
+            new_val = add_relative_normal_noise(pval, rel_stddev)[0]
+            setattr(self.params, pname, new_val)
+
+    def params_with_noise(self):
+        """Currently we add noise to a subset of params"""
+        return {
             f.name: getattr(self.params, f.name)
             for f in dataclasses.fields(self.params)
             if (
                 f.type == typing.Union[float, str]
                 and f.name not in ["Cglc_feed", "Cgln_feed"]
             )
         }
-        for pname, pval in float_params.items():
-            new_val = add_relative_normal_noise(pval, rel_stddev)[0]
-            setattr(self.params, pname, new_val)
 
     def execute(
         self,
         initial_conditions: typing.Optional[typing.Dict[str, typing.Any]] = None,
         plot: bool = False,
         sampling_stddev: float = 0.05,
+        starting_at_day: int = 0,
```

Comment:
why not hours, or setting to float?

Reply:
I thought we are indexing everything on a day basis (for e.g. params.Ndays), it might be more logical to keep starting at day level granularity.

Reply:
true, but it feels like a constraint that is not necessary (what if a process doesn't end on neat 24 h intervals?) I do think `int` is fine for now, especially if insilicho is only limited to streamlit use case